code/modules/mob/living/carbon/xenomorph/Xenomorph.dm

```diff
@@ -351,7 +352,7 @@
 
 	mutators.xeno = src
 
-	update_icon_source()
+	update_icon_source() //I'm not sure why this is here. Line 379 calls recalculate everything which calls this.
```

Comment:
please remove this comment or word it differently

line number will change as the file is modified in the future

Reply:
reworded comment to better explain what is happening